web/blueprints/finance/__init__.py

```diff
@@ -412,81 +411,106 @@ def bank_account_activities_edit(activity_id):
 @bp.route('/bank-account-activities/match/')
 @access.require('finance_change')
 def bank_account_activities_match():
-    FieldList = [
-        # ("Field-Name",BooleanField('Text')),
-    ]
+    matching_user, matching_team = match_activities()
+
+    field_list_user, matched_activities_user = _create_field_list_and_matched_activities_dict(matching_user,
+                                                                                              "user")
+    field_list_team, matched_activities_team = _create_field_list_and_matched_activities_dict(matching_team,
+                                                                                              "team")
```

Comment:
This would be one of the scenarios where an escape line-continuation would help:

```suggestion
    field_list_user, matched_activities_user \
        = _create_field_list_and_matched_activities_dict(matching_user, "user")
    field_list_team, matched_activities_team \
        = _create_field_list_and_matched_activities_dict(matching_team, "team")
```

Although a concise name would be even better.